dev-docs/best-practices/changelog.md

```diff
@@ -22,19 +22,34 @@ Each file is a markdown file with the following format:
 ```
 # (required) the semver level of this change, one of 'patch', 'minor', or 'major'
 level: ..
+# (required) the audience who this is applicable to, one of 'general', 'admins', 'deployers', 'users', or 'developers' (definied below)
+audience: ..
 # (optional) a reference to the bug or issue tracking this change ("bug xxxxxxx" or "issue xxx")
 reference:  ...
 ---
 <Snippet Text>
 ```
 
+## Audiences
+
+Taskcluster has several groups of people who interact with it:
+
+__users:__ people creating tasks (for testing or CI)
+
+__admins:__ people administering the service (managing scopes and roles)
+
+__deployers:__ people deploying the service (running TC services, setting service config values)
+
+__developers:__ people who are working on TC itself, making code changes, etc
```

Comment:
There's a similar list in the [docs best-practices](https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/docs.md#audience), added at @ajvb's suggestion, but it has five items.  Notably, I think "worker deployers" is an important category to add to the changelog, as worker upgrades tend to happen on a different cadence than services upgrades.  I'd love to have a less clunky name than "worker deployers" :)

Reply:
I'll just go with the clunky name. I've updated the changelog docs to use the docs-docs audiences and added a note the the docs-docs audiences to please update the changelog audiences when you change those.